db/db.go

```diff
@@ -39,6 +40,12 @@ func WithFlagOverride(ov map[string]string) Option {
 	}
 }
 
+func WithCustomOptions(co map[string]interface{}) Option {
```

Comment:
That's not the best. It's not typed. I believe in this case we want to create a type `minioOptions` that has typed, dedicated options.

Ideally something that also supports passing `func WithImage(image string) Option {` 🤔

Reply:
Changed it in https://github.com/efficientgo/e2e/pull/27/commits/0a6e650a80ad61756ece74ef15f3f6e986760654. Let me know if that works! :)